app/_locales/en/messages.json

```diff
@@ -418,6 +418,12 @@
   "advancedGasPriceTitle": {
     "message": "Gas price"
   },
+  "advancedPermissionSmallCase": {
```

Comment:
If "SmallCase" is the same as "Lowercase", I think "Lowercase" is easier to understand and consistent with other keys